Sources/xcodeproj/Objects/Targets/PBXProductType.swift

```diff
@@ -4,6 +4,7 @@ public enum PBXProductType: String, Decodable {
     case none = ""
     case application = "com.apple.product-type.application"
     case framework = "com.apple.product-type.framework"
+    case staticFramework = "com.apple.product-type.framework.static"
```

Comment:
Would you mind adding a comment explaining why this type and a reference to the project you mentioned on the PR description? I think we should clearly state that this is not a standard product type.